lib/src/js/compile.dart

```diff
@@ -182,6 +185,23 @@ OutputStyle _parseOutputStyle(String? style) => switch (style) {
 /// Converts [importer] into an [AsyncImporter] that can be used with
 /// [compileAsync] or [compileStringAsync].
 AsyncImporter _parseAsyncImporter(Object? importer) {
+  if (importer is NodePackageImporterClass) {
+    if (isBrowser) {
+      jsThrow(JsError(
+          "The Node Package Importer cannot be used without a filesystem."));
```

Comment:
"Node Package Importer" isn't a proper name, so either refer to the identifier (`NodePackageImporter`) or write it using normal case conventions ("Node package importer"). Also below.

Reply:
Addressed in https://github.com/oddbird/dart-sass/commit/15e33b9b2fc0a9848076f2356a1c99680f8198bc